wagtailio/project_styleguide/templates/patterns/components/streamfields/cards/related_page_card_block.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<a href="{{ value.url }}" class="logo-card {% if classes %}{{ classes }}{% endif %}">
+    {% include "patterns/components/meta/meta.html" with classes="logo-card__meta" %}
+
+    <h4 class="logo-card__heading teaser-heading">{{ value.title }}</h4>
+
+    <div class="logo-card__description">{{ value.introduction }}</div>
```

Comment:
Question: Is this a required field or should it be wrapped in a conditional?

Reply:
The `introduction` is a required field:

https://github.com/wagtail/wagtail.org/blob/ce622774084d59d669ed8a49adee0839312d7488/wagtailio/blog/models.py#L122